src/views/SessionView/Whiteboard.vue

```diff
@@ -2,6 +2,7 @@
   <div class="zwib-wrapper" :class="toolClass">
     <div id="zwib-div"></div>
     <div id="toolbar" class="toolbar">
+      <p v-if="error" class="error">{{ error }}</p>
```

Comment:
This error message is kind of just floating above the toolbar and is set to disappear after 2 seconds - it shows the user that they have uploaded a photo larger than 10mb. Can remove if unnecessary